olca-app-html/src/home/home.css

```
@@ -30,7 +30,7 @@ body {
     font-family: 'TheSans', Helvetica, Arial, sans-serif;
     color: #fbfbfb;
     line-height: 1.42857143;
-    background-image: url('images/background_start_page.jpg');
+    background-image: url('images/background.jpg');
```

Let's keep the same name. I guess Michael used that name to avoid confusion with any other HTML pages within openLCA.
```suggestion
    background-image: url('images/background_start_page.jpg');
```